app/graph/types/team_type.rb

```diff
@@ -1,118 +1,121 @@
 TeamType = GraphqlCrudOperations.define_default_type do
   name 'Team'
-  description 'Team type'
+  description 'The team (workspace) where annotation activity occurs.'
 
   interfaces [NodeIdentification.interface]
 
-  field :archived, types.Boolean
-  field :private, types.Boolean
-  field :avatar, types.String
-  field :name, !types.String
-  field :slug, !types.String
-  field :description, types.String
-  field :dbid, types.Int
-  field :members_count, types.Int
-  field :projects_count, types.Int
-  field :permissions, types.String
-  field :get_slack_notifications_enabled, types.String
-  field :get_slack_webhook, types.String
-  field :get_slack_channel, types.String
-  field :get_embed_whitelist, types.String
-  field :get_report_design_image_template, types.String
-  field :get_status_target_turnaround, types.String
-  field :pusher_channel, types.String
-  field :search_id, types.String
-  field :search, CheckSearchType
-  field :check_search_trash, CheckSearchType
-  field :trash_size, JsonStringType
+  field :archived, types.Boolean # TODO Rename to 'is_archived'
+  field :private, types.Boolean # TODO Rename to 'is_private'
+  field :avatar, types.String, 'Picture' # TODO Rename to 'picture'
+  field :name, !types.String, 'Team name'
+  field :slug, !types.String, 'Team slug (URL path)'
+  field :description, types.String, 'Team description'
+  field :members_count, types.Int, 'Count of team members'
+  field :projects_count, types.Int, 'Count of team projects (lists)'
+  field :get_slack_notifications_enabled, types.String # TODO Rename to 'slack_notifications_enabled'
+  field :get_slack_webhook, types.String # TODO Rename to 'slack_webhook'
+  field :get_slack_channel, types.String # TODO Rename to 'slack_channel'
+  field :get_embed_whitelist, types.String  # TODO Review and/or rename to 'embed_whitelist'
+  field :get_report_design_image_template, types.String # TODO Merge into 'report_settings'
+  field :search_id, types.String # TODO What's that for?
```

Comment:
Relay updates _still_ need a `search_id` - at least until we refactor `CheckSearch` to be real GraphQL